.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "files.insertFinalNewline": true
+}
```

Comment:
👻

Reply:
In case it's not obvious, this was intentional: by making the setting a workspace setting, you'll never have to tell anyone the same comment. 😅

Reply:
If you're complaining that the file has no terminating newline... That's how VSCode does it. 😅

Reply:
It just doesn't include the final newline in the settings file yet. :)

Reply:
VSCode!!!🙀